# These are the device numbers of the board for the linear actuators and
# rack and pinion motor.
emcee:
    daisy:
        linear_actuators:
            arm_left: 0
            arm_right: 1
            bucket_left: 2
            bucket_right: 3
        pinion: 4

        # I wasn't sure if the rack and pinion is using 1 or 2
        # pinions:
        #     left: 4
        #     right: 5
